docs/operator-manual/declarative-setup.md

```diff
@@ -431,6 +431,7 @@ The secret data must include following fields:
 
 * `name` - cluster name
 * `server` - cluster api server url
+* `namespaces` - optional list of namespaces which are accessible in that cluster. Cluster level resources would be ignored if namespace list if not empty. 
```

Comment:
Just spotted a little typo:
> Cluster level resources would be ignored if namespace list if not empty
should be (imho)
> Cluster level resources would be ignored if namespace list is not empty

Reply:
Thanks for spotting the typo, was a copy from the code documentation. It has been fixed for all occurrences in https://github.com/argoproj/argo-cd/pull/3992/commits/4d7daf572a79782051efcf78e20334e6734fc2b3.